### PR TITLE
Speed up CI tests

### PR DESCRIPTION
**Pull request by vicsn**
## Motivation

`synthesizer-program-integration` was still taking 3x as long as any other test suite in CI. This should resolve it.

## Test Plan

Would you mind reviewing @d0cd and seeing if you agree the # of iterations are still a good enough test?

## Related PRs

### CODE REVIEW

**Review by howardwu — APPROVED** [review]

**Review by Meshiest — APPROVED** [review]
Most of the changes in here are already present in the AleoNet mainnet-staging branch so I'm assuming the only real changes are the `synthesizer-program-integration-instruction-is`, `synthesizer-program-integration-instruction-equal`, and `synthesizer-program-integration-instruction-commit` jobs, which are passing and the `synthesizer-integration` pipeline job does seems to be running 3x faster (from 49min to 15min)

image

File: console/network/src/lib.rs [hunk 1]
```diff
@@ -91,7 +91,7 @@ pub trait Network:
     const INCLUSION_FUNCTION_NAME: &'static str;
 
     /// The fixed timestamp of the genesis block.
-    const GENESIS_TIMESTAMP: i64 = 1696118400; // 2023-10-01 00:00:00 UTC
+    const GENESIS_TIMESTAMP: i64;
```
Comment (gluax) [inline console/network/src/lib.rs:94]: Is this dead code now? Especially with the two `GENESIS_TIMESTAMP` below?

File: console/network/src/mainnet_v0.rs [hunk 2]
```diff
@@ -133,6 +133,8 @@ impl Network for MainnetV0 {
 
     /// The network edition.
     const EDITION: u16 = 0;
+    /// The fixed timestamp of the genesis block.
+    const GENESIS_TIMESTAMP: i64 = 1696118400 /* 2023-10-01 00:00:00 UTC */;
```
Comment (gluax) [inline console/network/src/mainnet_v0.rs:137]: i.e. here.

File: console/network/src/testnet_v0.rs [hunk 3]
```diff
@@ -136,6 +136,8 @@ impl Network for TestnetV0 {
     const GENESIS_COINBASE_TARGET: u64 = (1u64 << 5).saturating_sub(1);
     /// The genesis block proof target.
     const GENESIS_PROOF_TARGET: u64 = 1u64 << 3;
+    /// The fixed timestamp of the genesis block.
+    const GENESIS_TIMESTAMP: i64 = 1715776496 /* 2024-05-15 12:34:56 UTC */;
```
Comment (gluax) [inline console/network/src/testnet_v0.rs:140]: And here

**Review by gluax — APPROVED** [review]

File: console/network/src/lib.rs (hunk 1 above)
Comment (vicsn) [reply console/network/src/lib.rs]: This code in untouched, my bad I rebased incorrectly. Please look at the new diff

**Comment by vicsn** [comment]
Closing to merge into AleoNet

**Pull request closed without merging**